query/src/catalogs/backends/impls/meta_remote.rs

```diff
@@ -41,6 +41,7 @@ use common_meta_types::UpsertTableOptionReq;
 use crate::common::MetaClientProvider;
 
 /// A `MetaApi` impl with MetaApi RPC.
+#[allow(dead_code)]
```

Comment:
Why add so many dead_code here?

Reply:
Simply to avoid breaking the original design. :smile: